src/interfaces/models.ts

```diff
@@ -1,5 +1,7 @@
 import { EntityType, Location, NrState, RequestCode } from '@/enums'
 
+export type AnalysisJurisdictionT = 'BC' | 'XPRO'
+export type AnalysisTypeT = 'designation' | 'structure'
```

Comment:
I replaced other instances of these in my current ticket. These suck, because you can't use "constants" (enums) to assign or compare their values -- you're back to comparing strings. So I'd prefer to see enums for these.

Reply:
sure thing

Reply:
Hooray!